component-samples/demo/aio/aio.env

```diff
@@ -0,0 +1,54 @@
+aio_port=8080
+aio_database_connection_url=jdbc:h2:tcp://localhost:8081/./aio
+aio_database_username=sa
+aio_database_password=
+aio_database_port=8081
+aio_database_driver=org.h2.Driver
+aio_database_allow_others=true
+aio_database_init_sql=./init.sql
+aio_database_new_device_sql=./new-device.sql
+
+#epid_online_url=https://verify.epid-sbx.trustedservices.intel.com/
+#epid_test_mode=false
+catalina_home=./
+
+owner_keystore=./owner_keystore.p12
+owner_keystore_password=OnrKstr1
+owner_keystore_type=PKCS12
+
+#the keys provided to manufacturing or reseller
+owner_transfer_keys=owner_customer1.pem
+#owner keys  replacement during To2
+owner_replacement_keys=owner_customer1.pem
+
+# manufacturer_keystore can be blank when using pcks11
+manufacturer_keystore=./manufacturer_keystore.p12
+manufacturer_keystore_type=PKCS12
+manufacturer_keystore_password=MfgKs@3er
+
+#onDieChace directory must be absolute
+ondie_cache=file:///home/aio/onDieCache/
+ondie_autoupdate=false
+ondie_zip_artifact=
+ondie_check_revocations=true;
+
+aio_to0_max_ws = 28800
+aio_to0_scheduling_enabled=true
+aio_to0_scheduling_interval=300
```

Comment:
In AIO, can the to0 scheduling interval be 20 or 30 seconds? We are always changing the to0 scheduling interval value to 20 or 30.